app/src/main/java/com/mathewsachin/fategrandautomata/accessibility/ScriptRunnerService.kt

```diff
@@ -230,28 +253,40 @@ class ScriptRunnerService : AccessibilityService() {
 
     override fun onInterrupt() {}
 
+    private fun isLandscape() =
+        userInterface.metrics.let { it.widthPixels >= it.heightPixels }
+
+    override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
I don't remember, is there an active event filter for this similar to:
```
<activity android:name=".MyActivity"
          android:configChanges="orientation|screenSize|screenLayout|keyboardHidden"
          android:label="@string/app_name">
```

Reply:
That is for Activities. Activities are destroyed and recreated by default on configuration changes. That attribute can be set to tell Android not to destroy an activity on config changes and instead call the handler.
Services are not recreated on config changes so there's no manifest attribute to set for them.

Reply:
So what kind of configuration changes are you listening for? All of them?

Reply:
doesn't matter if I listen or not, that function is called for all config changes.